.bazelrc

```diff
@@ -21,6 +21,8 @@ fetch:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
 query:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
 sync:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
 
+test:oracle --test_env DAML_ORACLE_TESTING=true --test_env ORACLE_USERNAME --test_env ORACLE_PORT --test_env ORACLE_PWD
```

Comment:
The implementation in [#9511](https://github.com/digital-asset/daml/pull/9511/files#diff-47c2e72d225eae6ee56a0571dcf684365efaf380a7563daddaecb8283c597ce7R12) reads `DAML_ORACLE_TESTING` in a repository rule. So, I would expect that this requires `--repo_env` similar to `scala_2_13` is defined above.

Reply:
The tests run correctly, so what should break?

Reply:
#9511 sets `tags = ["manual"]` if `DAML_ORACLE_TESTING` doesn't read `"true"` in that repository rule. So, I would expect `bazel test //...` to not run this test-case if `DAML_ORACLE_TESTING` is not set globally or via `--repo_env`. IIRC `--test_env` does not affect repository rules.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9515/commits/cb7ec16d5d1e43364b80798f18e85cb5f5977a20

Reply:
Thanks!